huaweicloud/resource_huaweicloud_gaussdb_mysql_instance.go

```diff
@@ -201,6 +203,8 @@ func resourceGaussDBInstance() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 			},
+			"tags": tagsSchema(),
```

Comment:
add tags to docs

Reply:
as discussed, suggest to add a note here

Reply:
done